src/navs/program.js

```diff
@@ -10,6 +10,7 @@ export const programsNav = {
   Introduction: [
     pages['print-an-integer'],
     pages['add-two-integers'],
+    pages['multiply-two-numbers'],
```

Comment:
Add this line to the last in this list, 
You must follow the order of task list #201